skipchain/api_test.go

```diff
@@ -23,10 +23,7 @@ func TestClient_CreateGenesis(t *testing.T) {
 	_, roster, _ := l.GenTree(3, true)
 	defer l.CloseAll()
 	c := newTestClient(l)
-	_, cerr := c.CreateGenesis(roster, 1, 1, VerificationNone,
-		[]byte{1, 2, 3}, nil)
-	require.NotNil(t, cerr)
-	_, cerr = c.CreateGenesis(roster, 1, 0, VerificationNone,
+	_, cerr := c.CreateGenesis(roster, 1, 0, VerificationNone,
```

Comment:
I was expecting to see one test of each kind. Instead, it seems like you removed the previous test for []byte{1,2,3}. Why?